### PR TITLE
Updating Arabic Locale folder for OJS3

### PR DESCRIPTION
**Pull request by vormia**
These are the necessary changes in Arabic [local](https://github.com/vormia/ojs/tree/master/locale) folder so as to retrieve Arabic language to OJS3.
This is the 1st pull request. Others coming soon to complete all changes for adding Arabic back to OJS3.

### CODE REVIEW

File: lib/pkp [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 5668a7c205f7c73957f916e518fdb38063efa9b0
+Subproject commit 2e2ad168d15c14348d06bebb69a86ec633f4287d
```
Comment (mtub) [inline lib/pkp]: this is reverting the submodule link to a rather old version of pkp-lib

File: locale/ar_IQ/default.xml [hunk 2]
```diff
@@ -131,16 +131,11 @@
 	<pre>4. FORMATTING
```
Comment (mtub) [inline locale/ar_IQ/default.xml]: There's still some English text left, maybe you want to replace this as well?

File: locale/ar_IQ/emailTemplates.xml [hunk 3]
```diff
@@ -0,0 +1,852 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE email_texts SYSTEM "../../lib/pkp/dtd/emailTemplateData.dtd">
+
+<!--
+  * locale/ar_IQ/emailTemplates.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  * Localization information: https://pkp.sfu.ca/wiki/index.php?title=Translating_OxS
```
Comment (mtub) [inline locale/ar_IQ/emailTemplates.xml]: change link to "https://pkp.sfu.ca/wiki/index.php?title=OJS:_Arabic_(ar_IQ)"

File: plugins/importexport/duracloud/lib/DuraCloud-PHP [hunk 4]
```diff
@@ -0,0 +1 @@
+Subproject commit 31a1192887b7ff70a058ead1353f49be7246dccf
```
Comment (mtub) [inline plugins/importexport/duracloud/lib/DuraCloud-PHP]: @asmecher , do we need this?

**Comment by vormia** [comment]
Hi @mtub,
Glad to here from you,
In fact, I am confused here. Commits you see dating back to 2015 are my first commits to the **master** branch in my _repo_ when I tried pushing my translations for the first time.
I was struggling to clear it and bring it back to its original state, but things got worse! As you know, Github treats a revert as another commit.
Also, some file conflicts appeared from no where.
Then, after using Git shell, the result is the mess you see now.
I think I need to cancel all these commits, then update my master branch according to [PKP/OJS](https://github.com/pkp/ojs)... to make a clean pull request again.

Could you please help me with that...

**Comment by mtub** [comment]
Hi @vormia , whatever works best for you - you could start from a clean copy of ojs3 code, copy your locale files and push this to a new repository and send a new pull request. Or you could ask me to do that for you, I will happily assist you.

We could then merge the changes and start from there.

**Pull request closed without merging**